src/main/pages/portable-workspaces/creating-factories.adoc

```diff
@@ -77,78 +83,83 @@ Policies are a way to send instructions to the automation engine about the numbe
-== Lifecycle Events
+== Lifecycle events
+
+The lifecycle of the workspace is defined by the following events:
+
+`onAppLoaded`::
+	Triggered when the IDE is loaded.
 
-The lifecycle of the workspace is defined with the following events:
+`onProjectsLoaded`::
+	Triggered when the workspace and all projects have been activated.
 
-* `onAppLoaded` : Triggered when the IDE is loaded.
-* `onProjectsLoaded` : Triggered when the workspace and all projects have been activated.
-* `onAppClosed` : Triggered when the IDE is closed.
+`onAppClosed`::
+	Triggered when the IDE is closed.
 
 Each event type has a set of actions that can be triggered. There is no ordering of actions executed when you provide a list; {{ site.product_mini_name }} will asynchronously invoke multiple actions if appropriate.
 
 [id="factory-actions"]
-== Factory Actions
+== Factory actions
 
-Below is the list of all possible actions which can be configured with your Factory.
+Following is the list of all possible actions that can be configured with your factory.
 
 *Run Command*::
   _Description:_ Specify the name of the command to invoke after the IDE is loaded. +
   _Associated Event:_ `onProjectsLoaded`
 *Open File*::
-  _Description:_ Open project files in the editor. Optionally define line to be highlighted. +
+  _Description:_ Open project files in the editor. Optionally, define the line to be highlighted. +
   _Associated Event:_ `onProjectsLoaded`
 *Open a Welcome Page*::
   _Description:_ Customize content of a welcome panel displayed when the workspace is loaded. +
   _Associated Event:_ `onAppLoaded`
 *Warm on Uncommitted Changes*::
-  _Description:_ Opens a warning popup when the user closes the browser tab with a project that has uncommitted changes. +
+  _Description:_ Opens a warning pop-up window when the user closes the browser tab with a project that has uncommitted changes. +
   _Associated Event:_ `onAppClosed`
 
-See the link:factories_json_reference.html#ide-customization[Factory JSON reference] to learn how to configure Factory actions.
+To configure factory actions, see link:factories_json_reference.html#ide-customization[Factory JSON reference].
 
 [id="find-and-replace"]
-== Find and Replace
+== Finding and replacing variables
 
-Sometimes you may not want to expose certain sensitive information in source code (passwords, URLs, account names, API keys etc). Factories make it possible to replace variables or placeholders with real values. Find and replace can be run as a *Run Command* during `onProjectsLoaded` event. You can use `sed`, `awk` or any other tools that are available in your workspace environment.
+Factories allow you to replace variables or placeholders with real values to expose sensitive information (passwords, URLs, account names, API keys) in the source code. To find and replace a value, you can use the `run` command during an `onProjectsLoaded` event. You can use `sed`, `awk` or other tools available in your workspace environment.
```

Comment:
You're changing the meaning. Orig. talks about **not** exposing secrets.
```suggestion
Factories make it possible to replace variables or placeholders in the source code -- used to avoid exposing sensitive information (passwords, URLs, account names, API keys) -- with real values. To find and replace a value, you can use the `run` command during an `onProjectsLoaded` event. You can use `sed`, `awk`, or other tools available in your workspace environment.
```